pep-8103.rst

```diff
@@ -142,7 +138,43 @@ Answer #1 - #N: ``Candidates from Candidates_ Section``
 Results
 =======
 
-- TBD
+Of 85 eligible voters, 67 cast ballots.
+
+The top five vote-getters are:
+
+* Pablo Galindo Salgado
+* Petr Viktorin
+* Thomas Wouters
+* Gregory P. Smith
+* Brett Cannon
+
+No conflict of interest as defined in PEP 13 were observed.
```

Comment:
Grammar: did you mean "no conflict**s** of  interest were observed" or "no conflict of interest **was** observed"?